src/DynamoPublish/Resource.resx

```diff
@@ -117,25 +117,76 @@
   </resheader>
-  <data name="AuthenticationErrorMessage" xml:space="preserve">
+  <data name="AuthenticationFailedMessage" xml:space="preserve">
+    <value>Authentication failed.</value>
+  </data>
+  <data name="AuthManagerNotFoundMessage" xml:space="preserve">
     <value>Authentication manager is not found. Please, provide correct authentication manager.</value>
   </data>
   <data name="BadXMLConfigurationFileMessage" xml:space="preserve">
     <value>Malformed configuration file.</value>
   </data>
+  <data name="CloseButtonTitle" xml:space="preserve">
+    <value>Close</value>
+  </data>
+  <data name="CopyLinkButtonTitle" xml:space="preserve">
+    <value>Copy Link</value>
+  </data>
   <data name="DynamoViewMenuItemPublishTitle" xml:space="preserve">
     <value>Publish Customizer...</value>
   </data>
+  <data name="FailedMessage" xml:space="preserve">
+    <value>Failed</value>
+  </data>
   <data name="PageErrorMessage" xml:space="preserve">
     <value>"Page" is not set. set. Please, set server page in configuration file.</value>
   </data>
   <data name="PortErrorMessage" xml:space="preserve">
     <value>"Port" is not set. Please, set port in configuration file.</value>
   </data>
+  <data name="ProvideAuthProviderMessage" xml:space="preserve">
+    <value>Couldn't connect to server with this authentication provider. Try to use another provider.</value>
+  </data>
+  <data name="ProvideWorskspaceDescriptionMessage" xml:space="preserve">
+    <value>Please, provide workspace description.</value>
+  </data>
+  <data name="ProvideWorskspaceNameMessage" xml:space="preserve">
+    <value>Please, provide workspace name.</value>
+  </data>
+  <data name="PublishButtonTitle" xml:space="preserve">
+    <value>Publish</value>
+  </data>
+  <data name="PublishViewDefaultDescriptionTitle" xml:space="preserve">
+    <value>enter description here</value>
+  </data>
+  <data name="PublishViewDefaultNameTitle" xml:space="preserve">
+    <value>defaults to file name</value>
+  </data>
+  <data name="PublishViewDescriptionTitle" xml:space="preserve">
+    <value>Description:</value>
+  </data>
+  <data name="PublishViewNameTitle" xml:space="preserve">
+    <value>Name:</value>
+  </data>
+  <data name="PublishViewShareLinkTitle" xml:space="preserve">
+    <value>Share Link:</value>
+  </data>
   <data name="PublishViewTitle" xml:space="preserve">
     <value>Publish Customizer</value>
   </data>
-  <data name="ServerErrorMessage" xml:space="preserve">
+  <data name="ReadyForPublishMessage" xml:space="preserve">
+    <value>Workspace is ready for publish.</value>
+  </data>
+  <data name="ServerNotFoundMessage" xml:space="preserve">
     <value>Server not found.</value>
   </data>
+  <data name="UnknownServerErrorMessage" xml:space="preserve">
+    <value>Error on the server side. Please, connect dynamo team for more information.</value>
```

Comment:
There was an error when trying to publish your workspace. Please report this message to the Dynamo team.